indico/modules/events/abstracts/util.py

```diff
@@ -222,17 +234,30 @@ def get_user_abstracts(event, user):
 
 def get_visible_reviewed_for_tracks(abstract, user):
     event = abstract.event
-    if abstract.can_judge(user, check_state=True) or user in event.global_conveners:
+    if (abstract.can_judge(user, check_state=True) or
+       event.can_manage(user, permission='convene_all', explicit_permission=True)):
         return abstract.reviewed_for_tracks
-    convener_tracks = {track for track in event.tracks if track.can_convene(user)}
+    convener_tracks = set(track for track in event.tracks
+                          if track.can_manage(user, permission='convene', explicit_permission=True))
     return abstract.reviewed_for_tracks & convener_tracks
 
 
 def _query_user_tracks(event, user):
+    criteria = [db.and_(TrackPrincipal.type == PrincipalType.user,
+                        TrackPrincipal.user_id == user.id,
+                        TrackPrincipal.has_management_permission(permission='ANY'))]
+    for group in user.local_groups:
```

Comment:
I think we need to support multipass groups as well here (which isn't great performance-wise), OR we explicitly disallow anything but users (and event roles) for the abstract-related ACLs